assets/vue/components/BlockedIncident.vue

```diff
@@ -40,27 +40,49 @@ export default {
     incident: {type: Object, required: true},
     incidentResults: {type: Object, required: true},
     updateResults: {type: Object, required: true},
-    groupFlavors: {type: Boolean, required: true}
+    groupFlavors: {type: Boolean, required: true},
+    groupNames: {type: String, required: true}
   },
   computed: {
     updateResultsGrouped() {
       if (!this.groupFlavors) return this.updateResults;
       const results = {};
+      const groupNamesList = this.groupNames.toLowerCase().split(',');
       for (const value of Object.values(this.updateResults)) {
         const {flavor} = value.linkinfo;
         const {version} = value.linkinfo;
         const {groupid} = value.linkinfo;
         const newkey = `${groupid}:${version}`;
-        if (!(newkey in results)) {
-          results[newkey] = {name: value.name, passed: 0, failed: 0, stopped: 0, waiting: 0};
-          results[newkey].linkinfo = value.linkinfo;
-          results[newkey].linkinfo.flavor = [];
+        if (groupNamesList.includes(value.name.toLowerCase()) || this.groupNames === '') {
+          if (!(newkey in results)) {
+            results[newkey] = {
+              name: value.name,
+              passed: 0,
+              failed: 0,
+              stopped: 0,
+              waiting: 0,
+              linkinfo: {...value.linkinfo, flavor: []}
+            };
+          }
+          results[newkey].linkinfo.flavor.push(flavor);
+          results[newkey].passed += value.passed || 0;
+          results[newkey].stopped += value.stopped || 0;
+          results[newkey].waiting += value.waiting || 0;
+          results[newkey].failed += value.failed || 0;
```

Comment:
```suggestion
          const res = (results[newkey] ||= {
            name: value.name,
            passed: 0,
            failed: 0,
            stopped: 0,
            waiting: 0,
            linkinfo: {...value.linkinfo, flavor: []}
          });
          res.linkinfo.flavor.push(flavor);
          res.passed += value.passed || 0;
          res.stopped += value.stopped || 0;
          res.waiting += value.waiting || 0;
          res.failed += value.failed || 0;
```

Reply:
Could you please sync with @kraih ? To my understanding he prefers without the additional variable, see https://github.com/openSUSE/qem-dashboard/pull/303#discussion_r1145024803

Reply:
Then you must have misunderstood me. The extra variable is more efficient and easier to read, so of course i'm in favour of that. 😁